LocationManager/INTULocationManager/INTULocationManager.m

```diff
@@ -551,6 +635,43 @@ - (void)updateWithMaximumDesiredAccuracy:(INTULocationAccuracy)maximumDesiredAcc
     }
 }
 
+/**
+ Sets the CLLocationManager desiredActivityType
+ */
+- (void)updateWithDesiredActivityType:(CLActivityType)desiredActivityType
+{
+    switch (desiredActivityType) {
+        case CLActivityTypeFitness:
+            if (self.locationManager.activityType != CLActivityTypeFitness) {
```

Comment:
Is it good practice to check if it's currently what you are about to set it? (maybe so that it doesn't fire up location services on the switch?)

(vs was I was expecting was just setting the activityType)

Reply:
I don't think there is any real reason to check it before hand. At this point you have already requested a new location reading and there is no MAX activity type. I'll go ahead and remove this redundant check.